release_notes/ocp-4-0-release-notes.adoc

```diff
@@ -0,0 +1,828 @@
+[id='ocp-4-1-release-notes']
+= {product-title} 4.1 release notes
+include::modules/common-attributes.adoc[]
+:context: release-notes
+
+toc::[]
+
+== Overview
+
+Red Hat {product-title} provides developers and IT organizations with a hybrid
+cloud application platform for deploying both new and existing applications on
+secure, scalable resources with minimal configuration and management overhead.
+{product-title} supports a wide selection of programming languages and
+frameworks, such as Java, JavaScript, Python, Ruby, and PHP.
+
+Built on Red Hat Enterprise Linux and Kubernetes, {product-title}
+provides a more secure and scalable multi-tenant operating system for today’s
+enterprise-class applications, while delivering integrated application runtimes
+and libraries. {product-title} enables organizations to meet security, privacy,
+compliance, and governance requirements.
+
+[[ocp-4-1-about-this-release]]
+== About this release
+
+Red Hat {product-title}
+(link:https://access.redhat.com/errata/RHBA-2019:0758[RHBA-2019:0758]) is now
+available. This release is based on OKD 4.1, and it uses Kubernetes 1.13. New
+features, changes, and known issues that pertain to {product-title}
+{product-version} are included in this topic.
+
+Red Hat did not publicly release {product-title} 4.0 and, instead, is releasing
+{product-title} {product-version} directly after version 3.11.
+
+{product-title} {product-version} is supported on Red Hat Enterprise Linux 8 and
```

Comment:
You have to use {op-system-first} for the control plane, or master, machines and can use either {op-system} or Red Hat Enterprise 7.6 for compute, or worker, machines.

You might want to mention the different installation options. You can install  {product-title} {product-version} with installer provisioned infrastructure on Amazon Web Services AWS) or user provided infrastructure on AWS, bare metal, or vSphere hosts. If you choose to use the IPI installation, the cluster provisions and manages all of the cluster infrastructure for you.

Reply:
Per https://jira.coreos.com/browse/OSDOCS-350, I think we need a note here, or wherever we wind up talking about the BYO RHEL nodes, that says something like:
```
[IMPORTANT]
====
Because only Red Hat Enterprise Linux version 7.6 is supported for compute machines, you must not upgrade the Red Hat Enterprise Linux compute machines to version 8.
====
```